x-pack/plugins/stack_connectors/public/connector_types/webhook/webhook_connectors.tsx

```diff
@@ -28,29 +31,207 @@ import {
   TextField,
   ToggleField,
   PasswordField,
+  FilePickerField,
+  CardRadioGroupField,
+  HiddenField,
 } from '@kbn/es-ui-shared-plugin/static/forms/components';
 import { fieldValidators } from '@kbn/es-ui-shared-plugin/static/forms/helpers';
 import type { ActionConnectorFieldsProps } from '@kbn/triggers-actions-ui-plugin/public';
+import { WebhookAuthType, SSLCertType } from '../../../common/webhook/constants';
 import * as i18n from './translations';
 
 const HTTP_VERBS = ['post', 'put'];
 const { emptyField, urlField } = fieldValidators;
+const VERIFICATION_MODE_DEFAULT = 'certificate';
```

Comment:
I think we should default to 'full' to follow the same pattern as the [config in `kibana.yml`](https://www.elastic.co/guide/en/kibana/current/alert-action-settings-kb.html#action-config-custom-host-verification-mode)